src/controller/GameController.ts

```diff
@@ -44,22 +33,34 @@ export class GameController {
      * Advances the game to the next turn. Returns any new events for that turn, or the game state signifying that the game's
      * narrative has concluded, and the player can review his choices.
      */
-    public nextTurn(): Event[] | GameState {
+    public nextTurn(): { 
+        endgame: Boolean;
+        gameState: GameState;
+        nextEvent: Event;
+    } {
```

Comment:
GameState is now always returned to the controller so the front-end can show players data on the gamestate

_An example of me being hacky to get things done with my limited understanding_